app/workers/find_and_schedule_for_deletion_suspended_accounts_worker.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+class FindAndScheduleForDeletionSuspendedAccountsWorker
```

Comment:
Can the worker be called `ScheduleSuspendedAccountsForDeletionWorker`?

Reply:
https://github.com/3scale/porta/pull/17#discussion_r229267467 😄